.github/workflows/docker-GA-images.yml

```diff
@@ -44,29 +43,24 @@ jobs:
     if: github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v3
 
-      -
-        name: 'Set up JDK 17'
+      - name: 'Set up JDK 21'
         uses: actions/setup-java@v1
         with:
-          java-version: 17
+          java-version: 21
```

Comment:
We still use jdk17 in branch-4.1. :)

Reply:
@deniskuzZ Kindly ping. Should we change this back?

Reply:
nice catch!